pkg/master/controller/crdregistration/crdregistration_controller.go

```diff
@@ -77,9 +77,9 @@ func NewAutoRegistrationController(crdinformer crdinformers.CustomResourceDefini
 			cast := obj.(*apiextensions.CustomResourceDefinition)
 			c.enqueueCRD(cast)
 		},
-		UpdateFunc: func(_, obj interface{}) {
-			cast := obj.(*apiextensions.CustomResourceDefinition)
-			c.enqueueCRD(cast)
+		UpdateFunc: func(oldObj, newObj interface{}) {
+			c.enqueueCRD(oldObj.(*apiextensions.CustomResourceDefinition))
```

Comment:
probably worth a comment why we're enqueuing both (to make sure we process removed versions correctly)